src/main/java/com/google/swarm/tokenization/DLPTextToBigQueryStreamingV2PipelineOptions.java

```diff
@@ -189,4 +190,33 @@ public String create(PipelineOptions options) {
   String getDLPParent();
 
   void setDLPParent(String parent);
+
+  class InputPollingFactory implements DefaultValueFactory<InputLocation> {
+    @Override
+    public InputLocation create(PipelineOptions options) {
+       if (((DLPTextToBigQueryStreamingV2PipelineOptions) options).getFilePattern().startsWith("gs://"))  
+        return InputLocation.GCS;
+       else if (((DLPTextToBigQueryStreamingV2PipelineOptions) options).getFilePattern().startsWith("s3://"))
+        return InputLocation.S3;
+       throw new IllegalArgumentException("Data needs to be ingested either from GCS Bucket or S3 Bucket");
+    }
+  }
```

Comment:
Can the pipeline currently process files from local file system?

Reply:
Fixed